<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mode_pidt_ta</class>
 <widget class="QWidget" name="Mode_pidt_ta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_mode_pidt_save_params">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>621</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Wheel 1 Control</string>
   </property>
   <widget class="QDoubleSpinBox" name="sb_w1_kp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_43">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>KP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_69">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>KI:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_w1_ki">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_70">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>KD:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_w1_kd">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_71">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Step:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_step_w1_kp">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_72">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Step:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_step_w1_ki">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_73">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Step:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_step_w1_kd">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mode_pidt_write_params_pid_whe1">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>51</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Write</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_change_mode_pidt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode PIDT</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>621</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Sync Control</string>
   </property>
   <widget class="QDoubleSpinBox" name="sb_sta_kp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_40">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>KP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_49">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>KI:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_sta_ki">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_50">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>KD:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_sta_kd">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_51">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Step:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_step_sta_kp">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_52">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Step:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_step_sta_ki">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_53">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Step:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_step_sta_kd">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mode_pidt_write_params_pid_sync">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>51</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Write</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Motor 0:</string>
   </property>
   <widget class="QLineEdit" name="txtb_motor0_speed">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Desired speed:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_33">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enc speed:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtb_enc0_ta_speed">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_speed0">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set speed</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_mode_pidt_load_params">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>621</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Wheel 0 Control</string>
   </property>
   <widget class="QDoubleSpinBox" name="sb_w0_kp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_41">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>KP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_56">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>KI:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_w0_ki">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_57">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>KD:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_w0_kd">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_58">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Step:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_step_w0_kp">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_59">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Step:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_step_w0_ki">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_60">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Step:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sb_step_w0_kd">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mode_pidt_write_params_pid_whe0">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>51</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Write</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_mode_pidt_write_params">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Write</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>390</y>
     <width>201</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Control</string>
   </property>
   <widget class="QLabel" name="label_34">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>VX:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtb_pidt_vx">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_35">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>W:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtb_pidt_w">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_control_enable">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Disabled</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Motor 1:</string>
   </property>
   <widget class="QLineEdit" name="txtb_motor1_speed">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Desired speed:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enc speed:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtb_enc1_ta_speed">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_speed1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set speed</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
